recipe/meta.yaml

```diff
@@ -28,17 +28,20 @@ requirements:
     - numpy
     - numpydoc
     - ophyd >=1.5.0
+    - pcdsdevices
     - pcdsutils
-    - pydm >=1.16.0
-    - pyqt >=5
+    - platformdirs
+    - pydm >=1.19.1
+    - pyqt
```

Comment:
I think this is wrong too, there's one optional import for PyQt5 but otherwise nothing else

Reply:
This one is actually required if you want a functional typhos
qtpy does not install any Qt bindings, it just defers to the installed one

Reply:
It should be `pyqt =5` as (if I recall correctly) that's what we settled on